src/main/java/org/jitsi/jicofo/xmpp/IqHandler.java

```diff
@@ -39,12 +40,12 @@
  * @author Pawel Domas
  * @author Boris Grozev
  */
-public class MeetExtensionsHandler
+public class IqHandler
```

Comment:
The name feels a little broad, does this handle all IQs? Or only a certain scope of them?

Reply:
It's a subset (e.g. it doesn't include jingle), but it's pretty broad (authentication, conference request, muting, and rayo). What do you suggest as a better name?

Reply:
Is there something that all those things (everything but jingle it sounds like?) falls under?